rhodium/classification.py

```diff
@@ -77,6 +77,9 @@ def __init__(self,
         else:
             x = pd.DataFrame(x).to_records(index=False)
             
+        # Convert names to str since numpy chokes on unicode field names
+        x.dtype.names = map(str, x.dtype.names)
```

Comment:
Thanks for spotting this incompatibility with Numpy.  However, it seems using `str` won't cover all conversions.  For example, in Python 2, `str` throws an error if it can't map a character:

```
>>> title = u"Klüft skräms inför på fédéral électoral große"
>>> str(title)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xfc' in position 2: ordinal not in range(128)
```

And in Python 3, it appears to leave it in Unicode:

```
>>> title = u"Klüft skräms inför på fédéral électoral große"
>>> str(title)
'Klüft skräms inför på fédéral électoral große'
```

Should we be using `unicodedata.normalize(...)` or `str.encode('ascii', 'ignore')` instead?

Source: https://stackoverflow.com/questions/1207457/convert-a-unicode-string-to-a-string-in-python-containing-extra-symbols